tt_eager/tt_dnn/op_library/moreh_bmm/moreh_bmm_op.hpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "tensor/tensor.hpp"
+#include "tt_dnn/op_library/operation.hpp"
+
+namespace tt {
+namespace operations {
+namespace primary {
+
+using namespace tt_metal;
+
```

Comment:
Why doesn't moreh_bmm have 'struct MorehBMM' like other operations?

Reply:
moreh_bmm is a subset of moreh_matmul.
So internally it calls moreh_matmul op.